testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -638,4 +640,65 @@ protected static InputStream httpsAwareConfigurationStream(InputStream input) th
         }
         return in;
     }
+
+    /**
+     * This static method can be used to change default {@ode WebDriver} to different one if the test is able to run
```

Comment:
You mentioned "static" method, but this method is not static? How about declare it "static"? and maybe also move to some utils class like DroneUtils?

Reply:
sure, that was supposed to be static. Mea culpa!